spec/observables/from-promise-spec.js

```diff
@@ -32,4 +108,25 @@ describe('Observable.fromPromise', function () {
       done();
     });
   });
+
+  it('should globally throw unhandled errors', function (done) {
+    var invoked = false;
+    process.on('uncaughtException', function (reason, p) {
+      if (invoked) {
+        return;
+      }
+      invoked = true;
+      expect(reason).toBe('fail');
+      done();
+    });
+
+    Observable.fromPromise(Promise.reject('bad'))
+      .subscribe(
+        done.fail,
+        function (e) {
+          expect(e).toBe('bad');
+          throw 'fail';
```

Comment:
this test (seems) can be applied any kind of observable, i.e 

`Observable.throw('a').subscribe(..., function(e) { throw 'x' }, ...)` 

which emits error. It looks like testing behavior of `Observer::error()`, got curious if `PromiseObservable` has specific differences to handle those.